### DISCUSSION TITLE
线性图

### DISCUSSION THREAD

**Discussion by earthaYan** [antvis/G2#4975] [Q&A] [answered]
##### Checklist

- [ ] `npm test` passes
- [ ] benchmarks are included
- [ ] commit message follows commit guidelines
- [ ] documents are updated

##### Description of change

是否有途径可以在x轴为时间的折线图中选中其中一段区域，然后其他的折线图可以联动自动选中那一段时间区域
![9BGK6AfOmQ](https://user-images.githubusercontent.com/25888900/232987789-a6737ef6-ae58-4e0b-b16f-90a60eb35504.jpg)

**Comment by hustcc** [comment, marked as answer]
@earthaYan 看看官网上的这个[交互 DEMO ](https://g2.antv.antgroup.com/examples/interaction/interaction/#focus-context)是否符合预期。

![focus-context](https://user-images.githubusercontent.com/49330279/236596564-4d91f578-35c3-4607-bb02-5f348983787d.gif)

**Reply by earthaYan** [reply]
这个我们尝试过，和需求不太符合，所以目前结论是说服产品砍掉了这个需求，准备等不忙的时候再研究研究
